packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -122,6 +124,13 @@ export function PanelChrome({
           </TitleItem>
         </Tooltip>
       )}
+      {loadingState === LoadingState.Loading && onCancelQuery && (
+        <Tooltip content="Cancel query">
+          <TitleItem className={dragClassCancel} data-testid="panel-cancel-query" onClick={onCancelQuery}>
+            <Icon name="times-circle" size="md" />
```

Comment:
Can we use this? 
![Screenshot from 2023-03-14 17-29-39](https://user-images.githubusercontent.com/10999/225261492-d679677e-d026-49d3-a1a7-4cb9a18dc34d.png)

Reply:
fixed :)